databroker/broker.py

```diff
@@ -1087,16 +1087,62 @@ def export_size(self, headers):
 
         return total_size * 1e-9
 
-    def fill_events(self, events_iterable, descriptors,
-                    fields=True, inplace=False):
+    def fill_events(self, events, descriptors, fields=True, inplace=False):
         """Fill a sequence of events
 
+        This method will be used both inside of other `DataBroker`
```

Comment:
All `DataBroker` should be `Broker`. `DataBroker` is the name of the old singleton instance.